src/sentry/integrations/github/repository.py

```diff
@@ -109,8 +111,13 @@ def _format_commits(self, client, repo_name, commit_list):
     def _get_patchset(self, client, repo_name, sha):
         """Get the modified files for a commit
         """
-        commit = client.get_commit(repo_name, sha)
-        return self._transform_patchset(commit["files"])
+        key = "get_commit:{}:{}".format(repo_name, sha)
```

Comment:
I don’t know if there’s any rule, but you might wanna md5 repo_name. It seems likely there’s no bounds on length or weird characters that might cause issues. 

Also, use unicode.format, not str.format.